decidim-core/app/forms/decidim/registration_form.rb

```diff
@@ -19,6 +20,7 @@ class RegistrationForm < Form
 
     validates :sign_up_as, inclusion: { in: %w(user user_group) }
     validates :name, presence: true
+    validates :nickname, presence: true, length: { in: Decidim::User.nickname_length_range }
```

Comment:
I did some method renaming and now we ended up being inconsistent with `validatable`. I think it's fine, our naming is better so we can make it consistent when we remove it.